host/tests/dsCompositeIn_L3_Tests/dsCompositeIn_test2_VerifyStatus_Callback.py

```diff
@@ -0,0 +1,196 @@
+        if url is not None:
+            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
+
+        #download test streams to device
+        url = self.testSetup.assets.device.test2_VerifyStatus_Callback.streams
+        if url is not None:
+            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
+            for streampath in url:
+                self.testStreams.append(self.deviceDownloadPath + "/" + os.path.basename(streampath))
+
+    def testCleanAssets(self):
+        """
+        Removes the assets copied to the dut.
+
+        Args:
+            None.
+        """
+        self.deleteFromDevice(self.testStreams)
+
+    def testRunPrerequisites(self):
+        """
+        Runs Prerequisite commands listed in test-setup configuration file on the dut.
+
+        Args:
+            None.
+        """
+
+        #Run test specific commands
+        cmds = self.testSetup.assets.device.test2_VerifyStatus_Callback.execute
+        if cmds is not None:
+            for cmd in cmds:
+                self.writeCommands(cmd)
+
+    #TODO: Current version supports only manual verification.
+    def CheckDeviceStatus(self, manual=False, port_type:str=0):
+        """
+        Verifies whether Composite Source Device connected or not
+
+        Args:
+            port_type (str) : CompositeIn port
+            manual (bool, optional): Manual verification (True: manual, False: other verification methods).
+                                     Defaults to other verification methods
+
+        Returns:
+            bool
+        """
+        if manual == True:
+            return self.testUserResponse.getUserYN(f"Check if CompositeIn source is connected to {port_type} and press Y:")
+        else :
+            #TODO: Add automation verification methods
+            return False
+
+    def testFunction(self):
+        """This function will test and verify the active status callback
+
+        Returns:
+            bool
+        """
+
+        # Download the assets listed in test setup configuration file
+        self.testDownloadAssets()
+
+        # Run Prerequisites listed in the test setup configuration file
+        self.testRunPrerequisites()
+
+
+        # Create the dsCompositeIn class
+        self.testdsCompositeIn = dsCompositeInClass(self.deviceProfile, self.hal_session)
+
+        self.log.testStart("test2_VerifyStatus_Callback", '1')
+
+        # Initialize the dsCompositeIn module
+        self.testdsCompositeIn.initialise()
+
+        # Loop through the supported ports
+        for port in self.testdsCompositeIn.getSupportedPorts():
+            self.log.stepStart(f'Select {port} Port')
+            self.log.step(f'Select {port} Port')
+
+            portstr = f"dsCOMPOSITE_IN_PORT_{port}"
+            result = self.CheckDeviceStatus(True, portstr)
+            self.log.stepResult(result,f'CompositeIn Device connected {result} on {portstr}')
```

Comment:
use log.step instead of stepResult .

Reply:
Step and step result of that... used similar like audio